pkg/common/configs/configwatcher.go

```diff
@@ -111,16 +111,19 @@ func (cw *ConfigWatcher) Run() {
 		quit := make(chan bool)
 		go func() {
 			for {
+				select {
+				case <-quit:
+					<-cw.soloChan
+					ticker.Stop()
+					return
+				default:
```

Comment:
Can we add a `break` for the default clause?
I guess they work the same way with or without break, but just want to make it an explicit jump out of select if that makes sense.

Reply:
agree for clarity, not for functionality

Reply:
Thanks for the review, make sense to me.